### ISSUE TITLE
FreeJ2ME collaboration

### ISSUE THREAD

**Issue by jan-kleks** [evgenyzinoviev/gravitydefied#4]
Hello!

Recently a new multi-platform open-source J2ME emulator called [FreeJ2ME (GitHub repo)](https://github.com/hex007/freej2me/) has been released. [And the emulation scene seems to be quite happy about this news](https://www.reddit.com/r/emulation/comments/722f8i/freej2me_emulator/). What's more, sooner or later it should become a part of RetroArch what will allow for adding many awesome features that RetroArch is known for.

Would you be interested in working with them in some way?

I will also leave this link here to the very important J2ME preservation discussion (to inspire you): [The state of j2me/midlet/jar emulation, and the in-danger software for the platform that needs to be preserved](https://www.reddit.com/r/emulation/comments/4o2er3/the_state_of_j2memidletjar_emulation_and_the/)

Regards

**Comment by gch1p** [comment]
Hi (@evgenyzinoviev is me and this is my new profile).
What do you mean by working with them? Our port is not emulated in any way, it's native Android app.

**Comment by jan-kleks** [comment]
Yeah, I know. I have just thought that if you liked original J2ME Gravity Defied, you might be interested in a new J2ME emulator. If not -- that's fine, and you can close this issue. :) I'm sorry if I bothered you.

Regards
